tapiro/core/src/main/scala/io/buildo/tapiro/AkkaHttpMeta.scala

```diff
@@ -14,18 +14,19 @@ object AkkaHttpMeta {
     akkaHttpEndpoints: List[Defn.Val],
     routes: Term,
   ) => {
-    val tapirEndpoints = q"val endpoints = $endpointsName.create[AuthToken]()"
+    val tapirEndpoints = q"val endpoints = $endpointsName.create[AuthToken](statusCodes)"
     val akkaHttpEndpointsName = Term.Name(s"${controllerName.syntax}AkkaHttpEndpoints")
     q"""
     package ${`package`} {
       ..${imports.toList.map(i => q"import $i._")}
       import sttp.tapir.server.akkahttp._
       import sttp.tapir.Codec.{ JsonCodec, PlainCodec }
+      import sttp.model.StatusCode
       import akka.http.scaladsl.server._
       import akka.http.scaladsl.server.Directives._
 
       object $akkaHttpEndpointsName {
-        def routes[AuthToken](controller: $controllerName[AuthToken])(..$implicits): Route = {
+        def routes[AuthToken](controller: $controllerName[AuthToken], statusCodes: String => StatusCode)(..$implicits): Route = {
```

Comment:
looks like default is missing here `= _ => StatusCode.UnprocessableEntity`

Reply:
Because it's already provided [here](https://github.com/buildo/retro/pull/190/files#diff-a459f6affed72107c0633ca40d5677ddR31).

As I understand this, the user is going to interact only with `...Http4sEndpoints`, not with `...Endpoints`. As one _interface_ is stacked on top of the other I don't think we should replicate the same default at different layers.

Reply:
`Http4s` and `AkkaHttp` modules are never used together, this needs to be the same as `Http4sMeta`

Reply:
Sorry, I didn't understood your first feedback. What I wrote is total blabbering then.

Fixed in b317f8043bab8eec5c5c68809484323aeb7f3f01